src/screens/Profiles/components/TabHeader/component.js

```diff
@@ -14,7 +14,7 @@ export const TabHeader = ({
 }) => (
   <View style={styles.wrap}>
     <TitleText color={contentDark ? 'dark' : 'lightGrey'}>{title}</TitleText>
-    {(actionTitle && actionOnPress) && (
+    {actionTitle && (
```

Comment:
try passing an empty string to the actionTitle, it should crash the app. exactly the same reason with the above